core/src/main/scala/org/apache/spark/sql/crossdata/catalog/CatalogChain.scala

```diff
@@ -72,6 +115,7 @@ private[crossdata] class CatalogChain private(val temporaryCatalogs: Seq[XDTempo
     temporaryCatalogs.foreach(_.saveView(viewIdentifier, logicalPlan, sql))
 
   // TODO throw an exception if there is no temp catalogs! Review CatalogChain
```

Comment:
We could use require or throw an exception. At least one catalog (temporary or persistent) is already required, but both shouldn't be enforced.